src/view/DecisionPanel.java

```diff
@@ -22,15 +23,17 @@ public DecisionPanel(StatsPanel statsPanel) {
 		decisionsPanel.setPreferredSize(new Dimension(200,40));
 		
 		decisions[0] = new JButton("Next Turn");
-		decisions[1] = new JButton("Redo");
-		decisions[2] = new JButton("Undo");
+		decisions[1] = new JButton("Undo");
+		decisions[2] = new JButton("Redo");

```

Comment:
yeah...there were only the next turn and redo buttons for me, want to show this change asap so didnt want to bother figuring out why

Reply:
so i just swapped them